### PR TITLE
Admin access to the openshift-storage namespace for MT-SRE

### PR DESCRIPTION
**Pull request by nb-ohad**
Signed-off-by: nb-ohad <mitrani.ohad@gmail.com>

### CODE REVIEW

File: deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mtsre-admins
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: admin
+    namespacesAllowedRegex: (^openshift-storage>$)
```
Comment (apahim) [inline deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml:10]: There's a leftover `>`

File: deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mtsre-admins
```
Comment (apahim) [inline deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml:4]: Can you include the addon id in the name? I believe this name has to be unique.

File: deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml (hunk 1 above)
Comment (nb-ohad) [reply deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml]: Done

File: deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml (hunk 2 above)
Comment (nb-ohad) [reply deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml]: Done

**Comment by apahim** [comment]
This looks good from the Managed Tenants SRE side. Approved.

**Comment by dofinn** [comment]
cc @wanghaoran1988 are you able to review this RE:backplane?

**Comment by wanghaoran1988** [comment]
@nb-ohad Could you run `make` locally to generate the deploy template and commit again? looks like the "auto generated commit" is not happen due to a CI problem.

**Comment by nb-ohad** [comment]
@wanghaoran1988 
I have pushed a second "make artifacts" commit as requested

**Pull request merged**